src/cms/views/feedback/region_feedback_list_view.py

```diff
@@ -0,0 +1,71 @@
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.views.generic import TemplateView
+
+from ...constants import feedback_emotions
+from ...decorators import region_permission_required
+from ...models import (
+    Region,
+    PageFeedback,
+    EventFeedback,
+)
+
+
+@method_decorator(login_required, name="dispatch")
+@method_decorator(region_permission_required, name="dispatch")
+class RegionFeedbackListView(PermissionRequiredMixin, TemplateView):
+    permission_required = "cms.view_feedback"
+    raise_exception = True
+    template_name = "feedback/region_feedback_list.html"
+
+    # pylint: disable=too-many-locals
+    def get(self, request, *args, **kwargs):
+        # current region
+        region = Region.get_current_region(request)
+
+        page_feedback = PageFeedback.objects.filter(
+            page_translation__page__region=region, is_technical=False
+        )
+        event_feedback = EventFeedback.objects.filter(
+            event_translation__event__region=region, is_technical=False
+        )
+        event_list_feedback = region.event_list_feedback.filter(is_technical=False)
+        offer_feedback = region.offer_feedback.filter(is_technical=False)
+        offer_list_feedback = region.offer_list_feedback.filter(
+            region=region, is_technical=False
+        )
+        search_result_feedback = region.search_result_feedback.filter(
+            is_technical=False
+        )
+        imprint_page_feedback = region.imprint_page_feedback.filter(is_technical=False)
```

Comment:
Hi, 
Thanks for your Sugestion Changes...
it means that is_technical=False means that there is no Feedback??

Reply:
No, `is_technical=False` means that the feedback is targeted at the content creators, not at the programmers.
If you go to https://integreat.app/testumgebung/de and click on the emoticons on the left side, you can leave feedback for a specific page. The feedback function also has a dropdown "Wofür ist das Feedback?" where you can also select either "Inhalte von Testumgebung" or "Technische Funktionen". When you select "Technische Funktionen", it will be saved as `is_technical=True`. So the idea behind this is that feedback to "Inhalte von Testumgebung" is relevant for the region managers who created the content and feedback to "Technische Funktionen" is relevant for us programmers (thus visible at the admin level, not at the region level).